lib/trento_web/views/cluster_view.ex

```diff
@@ -0,0 +1,45 @@
+defmodule TrentoWeb.ClusterView do
+  use TrentoWeb, :view
+
+  def render("clusters.json", %{clusters: clusters}) do
+    render_many(clusters, __MODULE__, "cluster.json")
+  end
+
+  def render("cluster.json", %{cluster: cluster}), do: cluster
+
+  def render("cluster_registered.json", %{cluster: cluster}) do
+    cluster
+    |> Map.from_struct()
```

Comment:
As in hosts, maybe having the `Map.from_struct` in the `cluster.json` view would look more logical. With the removal of the common things (meta, tags).
This way, we wouldn't be sending the `__meta__` tag in the `clusters.json` multi rendered element

Reply:
done